public/doc/openapi-3-v1.0.0.json

```diff
@@ -717,6 +717,66 @@
                 }
             }
         },
+        "/workflows/resolve": {
+            "post": {
+                "tags": [
+                    "Workflow"
+                ],
+                "summary": "Get a list of workflows ids",
+                "description": "", 
+                "operationId": "resolveWorkflows",
+                "requestBody": {
+                   "content": {
+                       "application/json": {
+                           "schema": {
+                               "type": "object",
+                               "properties": {
+                                   "tags": {
+                                       "type": "array",
+                                       "items": {
+                                           "$ref": "#/components/schemas/Tag"
+                                       }
+                                   }
+                               }
+                           }
+                       }
+                   },
+                   "description": "Objects need to resolve workflows",
+                   "required": true
+                },
+                "responses": {
+                   "201": {
+                       "description": "Success",
+                       "content": {
+                           "application/json": {
+                               "schema": {
+                                   "type": "array",
+                                   "description": "Workflow ids associated with tags",
+                                   "items": {
+                                       "type": "integer"
```

Comment:
@hsong-rh All ids need to be strings and not integers, it might also be beneficial to create an id field and just use a reference.
Also I think it should have a readOnly set to users can never update it.